decidim-core/app/presenters/decidim/user_group_presenter.rb

```diff
@@ -27,6 +27,10 @@ def can_follow?
       false
     end
 
+    def officialized?
+      false
```

Comment:
Is this supposed to be like this ? It seems that an usergroup cannot be officialized.

Reply:
Yes, a user group does not respond to officialized? but from some cells this method is delegated to the presented resource. This definition avoids errors trying to call to officialized? and balances the methods the different presenters respond to.